<!DOCTYPE HTML>
<!--
	Photon by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Absolute quantification of mtDNA</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->

		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="stylesheet" href="assets/css/custom.css" />
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
	</head>
	<body>
			<nav class="navbar navbar-default navbar-fixed-top navbar-inverse">
			  <div class="container-fluid">
			  <ul class="nav navbar-nav navbar-left">
			  	<li><a href="index.html#landing_page">
			  		<i class="fa fa-line-chart fa-2x" aria-hidden="true"></i>
			  		</a>
			  	<li>
			  </ul>
	          <ul class="nav navbar-nav navbar-right">
		        <li><a href="index.html"><span>INTRO<span></a></li>
		        <li><a href="background.html"><span>BACKGROUND<span></a></li>
		        <li><a href="model.html"><span>MODEL<span></a></li>
		        <li><a href="inference.html"><span>INFERENCE<span></a></li>
		        <li><a href="results.html"><span>RESULTS<span></a></li>
		        <li><a href="summary.html"><span>SUMMARY<span></a></li>
		        <li><a href="references.html"><span>REFERENCES<span></a></li>
			    <li><a href="glossary.html"><span>GLOSSARY</span></a></li>
		      </ul>
			  </div>
			</nav>

			<section id="zero" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>Results</h2>
							</header>
							<p>The inference method was performed on both experimental and in-silico simulated data. Below we discuss the performance of the inference by looking at indicative plots such as joint and marginal posteriors, trace plots or Bayesian credible intervals.</p>
						</div>
						<div class="6u 12u$(medium)">
							<h3>Experimental data</h3>
							<p>The experimental data consists of qPCR amplification curves in the form of arrays of fluorescence reads. Each amplification curve represents the observation data used to infer the initial quantity of a target analyte X<sub>0</sub> (number of mtDNA molecules in a single cell). The qPCR experiments are performed in parallel, with the contents of each target analyte being placed in a separate well.
							</p>
							<p>Each amplification curve from experimental data has a corresponding X<sub>0</sub> estimate obtained using the standard curve method. This value is used as an additional indicator for whether the inferred posterior distribution for X<sub>0</sub> is centered around to the true value of the parameter. However, this value obtained from the standard curve method is simply an estimate, and should not be treated as the true X<sub>0</sub>.</p>
						</div>
						<div class="6u$ 12u$(medium)">
							<h3>In-silico simulation data</h3>
							<p>We perform in-silico simulations to obtain an amplification curve given a parametrisation <i>θ = [X<sub>0</sub>, r, σ]</i>, a known fluorescence coefficient <i>α</i> (which is not inferred) and a number of qPCR cycles <i>n</i>. The simulated process starts with <i>X=X<sub>0</sub></i> molecules, and is updated by sampling from a Binomial distribution with probability <i>r</i>. The fluorescence reads <i>f<sub>1:n</sub></i> are generated by adding noise sampled from a Gaussian distribution with variance <i>σ</i> (see <a href="model.html">Model</a>).
							</p>
							<p>The advantage of the in-silico data is that it allows us to compare the means and posterior distributions of all parameters <i>X<sub>0</sub>, r, σ</i> with the true values using simulated data <i>f<sub>1:n</sub></i>.</p>
						</div>
					</div>
				</div>
			</section>

			<section id="one" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>Trade-off between <i>X<sub>0</sub></i> and the efficiency <i>r</i></h2>
							</header>
							<p>In the first instance, inference was performed on data from a single amplification curve. As discussed in <a href="inference.html">Inference</a>, the exponential phase is extracted from the amplification curve and only those fluorescence reads are used as observation data. For the first ~30 cycles, the background noise dominates over the fluorescence signal, which leaves only data from ~4-5 cycles being informative for the inference of <i>X<sub>0</sub></i> and <i>r</i>, before significant saturation occurs and the assumption of <i>r</i> being a fixed parameter is clearly unreasonable.
							</p>
							<p>We found that this lack of information caused degeneracy in the inference of <i>X<sub>0</sub></i> and <i>r</i> by inspecting the joint posterior of the two parameters (see below). The explanation for this phenomenon is that for a short fluorescence curve, a similar fluorescence curve can be obtained by increasing the efficiency <i>r</i> and lowering the quantity of the target analyte <i>X<sub>0</sub></i> (and vice-versa). However, with more data from the "take-off" part of the curve, there would be more information to distinguish between <i>X<sub>0</sub></i> and <i>r</i>.
							</p>
							<p>The solution to this problem was to perform parameter-pooling inference using the data from multiple wells, with the assumption that parameters <i>r</i> and <i>σ</i> are shared between different wells within the same experimental setting. The new parameter for inference became <i>θ = [X<sub>0</sub><sup>1</sup>, X<sub>0</sub><sup>2</sup>, X<sub>0</sub><sup>3</sup>, r, σ]</i> for joint inference on 3 wells. Using data from multiple fluorescence curves where <i>r</i> is constant results in a more accurate inference of <i>r</i>, which in turn restricts the X<sub>0</sub><sup>j</sup> parameters. We see that the width of the marginals is strongly constrained through this alternative model structure.
							</p>
						</div>
						<br/>
						<div class="6u 12u$(medium) important(medium)">
							<div id="10_03_x0_p_sing_results" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="6u$ 12u$(medium) important(medium)">
							<div id="10_03_x01_p_joint_results" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
					</div>
				</div>
			</section>

			<section id="two" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>Marginal posterior distributions from real data</h2>
							</header>
							<p>We first look at results for the data obtained from real qPCR experiments. The goal of the inference method is to perform sampling from the joint posterior distribution <i>P(θ | f<sub>1:n</sub>, α)</i>. The fluorescence coefficient <i>α</i> relates fluorescence intensity to molecule copy number and is determined prior to the inference, using sigmoid fitting on samples where the initial copy number X<sub>0</sub> is known. Sigmoid fitting was used instead of inference because these samples have large X<sub>0</sub> values, causing the process to behave deterministically (more details in <a href="model.html">Model</a>). The inference for <i>P(θ | f<sub>1:n</sub>, α)</i> produces a sequence of samples of <i>θ</i> that approximates the joint distribution. From this sequence we can extract samples for each component of <i>θ</i> (marginalise) and plot the marginalised posterior distributions. 
							</p>
							<p>
							The marginal posterior distributions are easy to visualize and encode all of our uncertainty in the parameter given the data. For the target analyte quantity <i>X<sub>0</sub></i> we additionally plot the estimate from the standard curve method on top of the posterior distribution. The plot below shows the posterior distributions obtained from joint inference using target analytes X<sub>0</sub><sup>1</sup>, X<sub>0</sub><sup>2</sup>, X<sub>0</sub><sup>3</sup> from 3 separate qPCR wells (253, 254, 255).
							</p>
							<div id="10_03_x0_all_marg_results" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="12u$ 12u$(medium)">
							<br/>
							<p>The plots below demonstrate the effect of additional fluorescence data on the marginal posteriors by showing in parallel the results from single-well and joint inference. It is visible how the spread of posteriors narrows with joint inference, better localising the parameter values. Additionally, the prior used for each parameter is plotted: uniform prior for <i>X<sub>0</sub></i>, <i>Beta(0.5, 0.5)</i> for <i>r</i> and Jeffrey's prior for <i>σ</i>.
						</div>

						<br/> 
						<div class="6u 12u$(medium)">
							<div id="10_03_x0_single_results" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="6u$ 12u$(medium)">
							<div id="10_03_x01_marg_results" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>

						<div class="6u 12u$(medium)">
							<div id="10_03_p_single_results" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="6u$ 12u$(medium)">
							<div id="10_03_p_marg_results" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<br/>

						<div class="6u 12u$(medium)">
							<div id="10_03_sigma_single_results" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="6u$ 12u$(medium)">
							<div id="10_03_sigma_marg_results" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<br/>
					</div>
				</div>
			</section>

			<section id="three" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>Marginal posteriors distributions from simulated data</h2>
							</header>
							<p>In order to evaluate the inference method, we simulate a qPCR experiment in-silico and generate fluorescence data. We do so by choosing a true parameter to be the mean <i>θ</i> inferred from real data. An amplification curve is generated according to the random branching process model, using this fixed <i>θ</i> (for more details, see <a href="model.html">Model</a>). For brevity, we only look at results from the joint inference, as it has been shown in the previous section to remove parametric degeneracy.
							</p>
							<div id="simulated_x0_marginal" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>

						<br/> 
						<div class="6u 12u$(medium)">
							<div id="simulated_p_marginal" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="6u$ 12u$(medium)">
							<div id="simulated_sigma_marginal" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
					</div>
				</div>
			</section>

			<section id="four" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>The stochasticity in amplification</h2>
							</header>
							<p>A main advantage of the Bayesian inference approach in qPCR absolute quantitation is that it accounts for the stochasticity of the amplification process, by modelling it as a random branching process with efficiency <i>r</i>. The stochasitcity of the process can be exhibited using Bayesian credible intervals (BCI).</p>
							<p>The inference algorithm produces a sequence of <i>θ</i> samples that approximate sampling from the posterior distribution of <i>θ</i>. A qPCR experiment is simulated in-silico for each sample, resulting in a fluorescence read sequence. A 5%-95% BCI for the fluorescence reads across all simulated experiments is determined at each cycle. The area plotted below shows the BCI for the fluorescence data (green). Additionally, the plot shows the real fluorescence data used for inference (red), as well as the mean fluorescence value at each cycle (blue).</p>
							</p>
							<p><i> NOTE! This is not what shows stochasticity! Shouldn't it be different simulations for the same parameter?</i></p>
							<div id="bci" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
							<div id="bci_zoomed" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
					</div>
				</div>
			</section>


			<section id="five" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>The effect of the fluorescence coefficient α on inference results</h2>
							</header>
							<!--
							<p>One of the main challenges of the project has been finding an appropriate way to treat the fluorescence coefficient α. Existing research [*quote] treats α as a fixed constant and do not discuss ways.
							</p>
							--><p>The inter- and intra-experimental variability of the fluorescence coefficient α was discussed in the <a href="model.html">Model</a> section and is showed again in the plot below. Estimates for α were obtained by fitting sigmoid curves on data from standards with known initial quantities X<sub>0</sub>. Sigmoid fitting was used instead of inference because these standards had a large copy number X<sub>0</sub>, causing the amplification process to behave deterministically. It has been discussed that the inter-experimental variability in α is explained by the use of different primers (experiments 3 and 4 below use the same primer and have a similar distribution). Thus the α estimate can be used for inference on a new sample, if the sample uses the same primer as the standards.</p>
							
							<div id="boxplot" style="min-width: 310px; height: 400px; margin: 0 auto"></div>

							<p>
							 We are now interested in how different values of α from the same qPCR experimental setting influence the posterior distribution of <i>X<sub>0</sub></i>.
							</p>
							<p>The experiments used to estimate α have the following initial standard quantities: 3 wells with 100,000 molecules, 3 with 10,000 and 3 with 1,000. The plots below show posteriors from inference runs with different α values, grouped by the initial number of molecules in the curve used to determine α.
							</p>

							<div id="different_alphas_posteriors_100000" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
							<div id="different_alphas_posteriors_10000" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
							<div id="different_alphas_posteriors_1000" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
							<p>The initial number of mtDNA molecules in single-cells are of the order of <i>100</i>, whereas standard experiments used to estimate α had more that <i>1,000</i> molecules. Recall the equation:</p>
							<p style="text-indent: 50px">
								<i>F<sub>i</sub> = αX<sub>i</sub> + N(0, σ<sup>2</sup>)</i>
							</p>
							<p>One might expect an improvement in the estimation of α as X values get smaller and do not dominate the fluorescence term, making it more sensitive to the coefficient α. Surprisingly however, the plots above indicate that the best posterior is obtained for the intermediate values of the initial standard quantity, not the smallest. It might be the case that for highly diluted standards, the initial number of standard molecules is too small to be determined correctly, so α is estimated incorrectly by using a wrong value of X<sub>0</sub> for the standard.
							<!-- small->more sensitive to alpha; too small->incorrect estimate?-->
							</p>
							<p>The inference results plotted in other sections of this page all use the mean value of α from estimates obtained with sigmoid-fitting.</p>
							</p>
						</div>
					</div>
				</div>
			</section>

			<section id="six" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>Analysing the Markov chain</h2>
							</header>
							<p>A Metropolis-Hastings algorithm should produce a Markov chain with the desired joint posterior <i>P(θ | f<sub>1:n</sub>, α)</i> as its stationary distribution. Two common ways of assessing the properties of the Markov chain are trace plots and autocorrelation function (ACF) plots. They indicate how many steps the algorithm should be run for, what the burnin period should be and how the covariance of proposal distribution Q(θ | θ') can be calibrated.</p>

							<h3>Trace plots</h3>
							<p>Incomplete - discussion needed </p>
							<!--<p>A trace plot shows the values of samples produced by the inference algorithm over the number of iterations. It provides information about some aspects of algorithm calibration: the number of iterations, the covariance of the proposal distribution or the burn-in. A well-calibrated algorithm forms a Markov-Chain of </p>
						-->
							<div id="mixing_x01" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="6u 12u$(medium)">
							<div id="mixing_p" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="6u$ 12u$(medium)">
							<div id="mixing_sigma" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="12u$ 12u$(medium)">
						<h3>Autocorrelation function (ACF)</h3>
						</div>
						<br/>
						<div class="4u 12u$(medium)">
							<div id="acf_x01" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="4u 12u$(medium)">
							<div id="acf_p" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
						<div class="4u$ 12u$(medium)">
							<div id="acf_sigma" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
						</div>
					</div>
				</div>
			</section>


		<!-- Footer -->
			<section id="footer">
				<ul class="actions">
					<li><a href="summary.html" class="button scrolly">Read next section</a></li>
				</ul>
				<ul class="copyright">
					<li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
				</ul>
			</section>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>
			<!-- High charts -->
			<script src="https://code.highcharts.com/highcharts.js"></script>
			 <script src="https://code.highcharts.com/highcharts-more.js"></script>
			<script src="https://code.highcharts.com/modules/exporting.js"></script>
			<script src="10_03_plots.js"></script>

	</body>
</html>